src/UnitTests/Constructors.cs

```diff
@@ -5,6 +5,175 @@
 
 namespace AutoMapper.UnitTests.Constructors
 {
+    public class When_constructor_is_partial_match_with_value_type : AutoMapperSpecBase
+    {
+        GeoCoordinate _destination;
+
+        public class GeolocationDTO
+        {
+            public double Longitude { get; set; }
+            public double Latitude { get; set; }
+            public double? HorizontalAccuracy { get; set; }
+        }
+
+        public struct GeoCoordinate
+        {
+            public GeoCoordinate(double longitude, double latitude, double x)
+            {

```

Comment:
Lol duh

On Wednesday, July 6, 2016, Lucian Bargaoanu <notifications@github.com>
wrote:

> In src/UnitTests/Constructors.cs
> <https://github.com/AutoMapper/AutoMapper/pull/1434#discussion_r69729218>:
>
> > @@ -5,6 +5,175 @@
> >
> >  namespace AutoMapper.UnitTests.Constructors
> >  {
> > +    public class When_constructor_is_partial_match_with_value_type : AutoMapperSpecBase
> > +    {
> > +        GeoCoordinate _destination;
> > +
> > +        public class GeolocationDTO
> > +        {
> > +            public double Longitude { get; set; }
> > +            public double Latitude { get; set; }
> > +            public double? HorizontalAccuracy { get; set; }
> > +        }
> > +
> > +        public struct GeoCoordinate
> > +        {
> > +            public GeoCoordinate(double longitude, double latitude, double x)
> > +            {
>
> It's a value type. It has a default constructor.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/AutoMapper/AutoMapper/pull/1434/files/f3df2541fcb1be1f7fb0345a2e1f55c5dc18ca30#r69729218>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGYMs6bPwJwa-o7Qj3415-2Eoxb-eVDks5qS62PgaJpZM4JENMf>
> .
>